package.json

```diff
@@ -6,7 +6,8 @@
     "install:frontend": "cd frontend && npm install",
     "install:backend": "cd backend && npm install",
     "start:frontend": "cd frontend && npm start",
-    "start:server": "cd backend && npm start"
+    "start:server": "cd backend && npm start",
+    "test:e2e": "start-server-and-test \"npm run start:server\" 3001 \"npm run start:frontend\" 3000 \"npx cypress open\""
```

Comment:
Esse comando não funciona visto que faltou instalar a biblioteca `start-server-and-test` como uma dependência de desenvolvimento, ou seja, não está listada nas `devDependencies`.